pkg/controller/podautoscaler/horizontal.go

```diff
@@ -337,7 +337,7 @@ func (a *HorizontalController) computeReplicasForMetrics(hpa *autoscalingv2.Hori
 		}
 	}
 
-	setCondition(hpa, autoscalingv2.ScalingActive, v1.ConditionTrue, "ValidMetricFound", "the HPA was able to succesfully calculate a replica count from %s", metric)
+	setCondition(hpa, autoscalingv2.ScalingActive, v1.ConditionTrue, "ValidMetricFound", "the HPA was able to successfully calculate a replica count from %s", metric)
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/commit/151a7d2731d3526e7d11e899ebc652c080db86fb. Closing.

/close